VCS/1. User Guides/14. 建立資料磁碟.md

```diff
@@ -1 +1,95 @@
+---
+tags: Guide, VCS, VDS, EN
+title: 建立資料磁碟 | en
+GA: UA-155999456-1
+---
 
+{%hackmd @docsharedstyle/default %}
+{%hackmd @docsharedstyle/twccheader-en %}
+
+# Create data disk
+
+If the storage space of the VCS instance is insufficient, you can create a data disk. Then, mount the disk on the instance, and initialize the disk for use.
+
+
+<!-- 1 start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC Portal</b></summary>
+
+<br>
+
+### 1. Create a data disk when creating VCS instance
+
+- You can create a data disk when creating the [<ins>VCS instance</ins>](https://man.twcc.ai/@twccdocs/guide-vcs-create-en). On **STORAGE** tab, create a data disk (can only create 1 disk, and only the disk type and data disk size can be decided, and the disk name cannot be decided).
+
+### 2. Create a data disk on Data Disk Management page
+
+If you need to create more disks, please follow the instructions below:
+
+* Click **Virtual Disk Service (VDS)** from the service list. On the **Data Disk Management** page, click **＋CREATE**.
+
+
+* Fill in the data disk name, description, whether the data disk is to be created newly or restored from the snapshot, size(GB), and disk type (HDD or SSD), and then click **NEXT: REVIEW & CREATE>**.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_30e5b2632e35e15bda8378a625e74a07.png)
+
+
+
+
+<!-- :::info
+<i class="fa fa-paperclip fa-20" aria-hidden="true"></i> **附註：** 選擇使用 SSD 加密資料磁碟可加強您的資料安全性，但加密過程將可能略微影響存取效率。
+::: -->
+
+
+* Review the configurations of the data disk and then click **CREATE**.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_3f58724e9c6d407f059cfcedb52165d3.png)
+
+
+</details>
+
+<!-- Space -->
+
+<div style="height:8px"></div>
+
+<!-- 2. start -->
+
+<details class="docspoiler">
+
+<summary><b>TWCC CLI</b></summary>
+
+<br>
+
+
+### Commands
+
+```bash
+$ twccli mk vcs [-n]   #Data disk name
+                [-sz]  #Data disk size (GB)
+                
+```
+:::info
+{%hackmd @twccdocs/cli-parameter-note-en %}
+2. If you do not enter the parameter of the disk, it will be created with the default settings:
```

Comment:
[SUG]

enter the parameter of the disk > specify the optional parameters, the disk will be created with default settings